.travis.yml

```diff
@@ -16,11 +15,6 @@ env:
 matrix:
     allow_failures:
         - php: 7.0
-    include:
-        - php: 5.3
-          env: TEST_SUITE=1 COMPOSER_FLAGS="--prefer-lowest"
-        - php: 5.3
```

Comment:
It will be discussed separately, but I think changing it to 5.4 is a must :wink: Anyway you can leave it for now.